<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>OrganizationPropDlg</class>
 <widget class="QDialog" name="OrganizationPropDlg">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>413</width>
    <height>211</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Company properties</string>
  </property>
  <property name="locale">
   <locale language="English" country="UnitedStates"/>
  </property>
  <property name="sizeGripEnabled">
   <bool>true</bool>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="label_Short">
       <property name="text">
        <string>Short name</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="leShortName"/>
     </item>
     <item>
      <widget class="QRoCheckBox" name="cbClient">
       <property name="text">
        <string>Client</string>
       </property>
       <property name="checked">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRoCheckBox" name="cbProvider">
       <property name="text">
        <string>Supplier</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRoCheckBox" name="cbDeleted">
       <property name="text">
        <string>Deleted</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QLabel" name="label_Full">
       <property name="text">
        <string>Full name</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="leFullName"/>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QTableWidget" name="twCustProps">
     <property name="frameShape">
      <enum>QFrame::NoFrame</enum>
     </property>
     <property name="selectionMode">
      <enum>QAbstractItemView::SingleSelection</enum>
     </property>
     <attribute name="horizontalHeaderVisible">
      <bool>false</bool>
     </attribute>
     <attribute name="horizontalHeaderStretchLastSection">
      <bool>true</bool>
     </attribute>
     <attribute name="verticalHeaderVisible">
      <bool>false</bool>
     </attribute>
     <column>
      <property name="text">
       <string/>
      </property>
      <property name="font">
       <font>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
     </column>
     <column>
      <property name="text">
       <string/>
      </property>
     </column>
    </widget>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>391</width>
       <height>23</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QRoCheckBox</class>
   <extends>QCheckBox</extends>
   <header>../VProject/QRoCheckBox.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>OrganizationPropDlg</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
